examples/llm/src/models/hf/hf_t5.py

```diff
@@ -81,13 +84,21 @@ def __init__(self, cfg: DictConfig):
                                  ignore_index=_HF_IGNORE_INDEX),
             MaskedAccuracy(ignore_index=_HF_IGNORE_INDEX)
         ]
+        eval_metrics = [
+            LanguageCrossEntropy(vocab_size=vocab_size,
+                                 ignore_index=_HF_IGNORE_INDEX),
+            MaskedAccuracy(ignore_index=_HF_IGNORE_INDEX),
+            InContextLearningLMAccuracy(),
+            InContextLearningMultipleChoiceAccuracy()
```

Comment:
These ICL eval tools don't support encoder-decoder yet. Maybe we should hold off on adding these until they are supported?

Reply:
ah, good call. will remove.